### PR TITLE
Added helper functions to database

### PR DESCRIPTION
**Pull request by kdreese**

### CODE REVIEW

File: database.js [hunk 1]
```diff
@@ -85,10 +81,48 @@ Task.init({
...
+}
+
+//createUser: login(username), peper(salt), browns(hash)
+function createUser(login, pepper, browns){
+    User.create({
+        salt: pepper,
+        hash: browns,
+        username: login,
+    });
+}
```
Comment (thearst3rd) [inline database.js]: `salt` -> `pepper`
`hash` -> `browns`

this is truely literary genius

**Comment by thearst3rd** [comment]
This all looks good to me

**Comment by tmwbook** [comment]
@kdreese looks good. Please add the helpers to an export statement somewhere

**Comment by tmwbook** [comment]
Oh, looks like we also need some sort of function to establish the connection to the database as well.

Edit: Last thing, I promise! Looks like we do have to install the sqlite npm package, we just never import it. See the [Sequelize Docs](https://sequelize.org/master/manual/getting-started.html).

**Pull request merged**